io.github.aldrinzigmundv.InnerMirror.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "io.github.aldrinzigmundv.InnerMirror",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "innermirror",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/rust-flatpak/cargo"
+        }
+    },
+    "modules": [
+        {
+            "name": "innermirror",
+            "buildsystem": "meson",
+            "run-tests": true,
+            "build-commands": [
```

Comment:
buildsystem is meson so build-commands is ignored.

Reply:
It actually should just work to use meson. You should even be able vendor the crates in the tarball and not need cargo-sources....